frontend/src/components/forms/ProfileEditForm.tsx

```diff
@@ -84,7 +88,7 @@ const ProfileEditForm: FunctionComponent<Props> = ({
     form.values.items,
     (v) => {
       const suffix =
-        v.hi === "True" ? ":hi" : v.forced === "True" ? ":forced" : "";
+        v.hi === "only" ? ":hi" : v.forced === "True" ? ":forced" : "";
```

Comment:
Not sure if intentional, but as mentioned on Discord, this will display:

- en (normal or hi / "also")
- en (normal / "never" )
- en:hi ("only")
- en:forced ("forced)